tools/challenge-editor/client/package.json

```diff
@@ -14,7 +14,7 @@
     "typescript": "4.7.4"
   },
   "scripts": {
-    "start": "PORT=3300 react-scripts start",
+    "start": "react-scripts start",
```

Comment:
As a heads up, I explicitly set the port here because React defaults to 3000 - which conflicts with our `/learn` client.
 
I wanted contributors to be able to run this editor to work on challenges while also running the actual learn platform so they could see/test their changes.